pep-0001.txt

```diff
@@ -517,7 +518,7 @@ optional and are described below.  All other headers are required. ::
     Title: <pep title>
     Author: <list of authors' real names and optionally, email addrs>
   * Sponsor: <real name of sponsor>
-  * BDFL-Delegate: <PEP czar's real name>
+  * PEP-Delegate: <PEP czar's real name>
```

Comment:
This is the only place that "czar" is still used in the PEP.  Maybe just say "PEP delegate's real name"?

Reply:
```suggestion
  * PEP-Delegate: <PEP delegate's real name>
```